src/Spryker/Zed/Customer/Business/Customer/Customer.php

```diff
@@ -505,6 +508,14 @@ public function restorePassword(CustomerTransfer $customerTransfer)
             return $customerResponseTransfer;
         }
 
+        $customerResponseTransfer = $this
```

Comment:
the test file should be also copied from the original PR (`Bundles/Customer/tests/SprykerTest/Zed/Customer/Business/Facade/RestorePasswordTest.php`).

Reply:
I thought we don't backport tests? 😕

Reply:
we backporting all diffs